src/controllers/user-management/auth.controller.ts

```diff
@@ -109,13 +109,14 @@ export class AuthController {
         // - nameIDFormat is used. Correspond to <NameIDFormat> in the public certificate
         reqConverted.samlLogoutRequest = null; // Property must be set, but it is unused in the source code
         // reqConverted.user.nameID = reqConverted.user.nameID;
-        reqConverted.user.nameIDFormat = "urn:oasis:names:tc:SAML:1.1:nameid-format:X509SubjectName";
+        reqConverted.user.nameIDFormat =
+            "urn:oasis:names:tc:SAML:1.1:nameid-format:X509SubjectName";
         // reqConverted.user = { reqCo };
         // TODO: Remove after test
-        this.logger.debug(`KOMBIT logout request: ${JSON.stringify(req)}`)
+        this.logger.debug(`KOMBIT logout request: ${JSON.stringify(req)}`);
 
         this.strategy.logout(reqConverted, (err: Error, url: string): void => {
-            req.logout();
+            req.logout(err1 => {});
```

Comment:
My eslint is marking this as an error: Unexpected empty arrow function. 
It still compiles though, so it should be okay. If the error should be removed, we cold replace {} with undefined.